arbitrator/config/arbitrator_params.yaml

```diff
@@ -28,4 +28,15 @@ use_fixed_costs: true
 # Map: The priorities associated with each plugin during the planning 
 # process, values will be normalized at runtime and inverted into costs
 # Unit: N/a
-plugin_priorities: {platoon_strategic_ihp: 4.0, lci_strategic_plugin: 3.0, sci_strategic_plugin: 2.0, platoon_strategic: 2.0, route_following_plugin: 1.0}
\ No newline at end of file
+plugin_priorities: '
+  {
+ 	  "plugin_priorities": 
+      {
+ 			  "platoon_strategic_ihp": 4.0,
```

Comment:
Is the indentation here accurate? It looks off for this line and the one above it.